py-polars/tests/parametric/test_lazyframe.py

```diff
@@ -16,25 +16,35 @@
             column(
                 "start",
                 dtype=pl.Int8,
-                null_probability=0.15,
-                strategy=integers(min_value=-4, max_value=6),
+                null_probability=0.3,
+                strategy=integers(min_value=-3, max_value=4),
             ),
             column(
                 "stop",
                 dtype=pl.Int8,
-                null_probability=0.15,
-                strategy=integers(min_value=-2, max_value=8),
+                null_probability=0.3,
+                strategy=integers(min_value=-2, max_value=6),
             ),
             column(
                 "step",
                 dtype=pl.Int8,
-                null_probability=0.15,
+                null_probability=0.3,
                 strategy=integers(min_value=-3, max_value=3).filter(lambda x: x != 0),
             ),
             column("misc", dtype=pl.Int32),
         ],
     )
 )
+@example(
```

Comment:
@alexander-beedie do we want to have this merged?

Reply:
May as well; can remove the other test later - once I've fixed `random` I can tidy it up.